src/rules/html/warn-on-remote-script.js

```diff
@@ -0,0 +1,25 @@
+import { VALIDATION_WARNING } from 'const';
+import { isStrictRelativeUrl } from 'schema/formats';
+import * as messages from 'messages';
+
+
+export function warnOnRemoteScript($, filename) {
+  return new Promise((resolve) => {
+    var linterMessages = [];
+
+    $('script').each((i, element) => {
+      const src = $(element).attr('src');
+
+      if (src !== undefined && !isStrictRelativeUrl(src)) {
+        linterMessages.push(
+          Object.assign({}, messages.REMOTE_SCRIPT, {
+            type: VALIDATION_WARNING,
```

Comment:
There are certainly cases where we have allowed remote scripts in the past, so it can't be a hard error. Otherwise, making an exception means an admin will have to upload every update in order to be accepted.